site/en/docs/extensions/mv3/mv3-migration/index.md

```diff
@@ -549,6 +549,26 @@ reference documentation for a more detailed description of rules
 This feature allows content blockers and other request-modifying extensions to implement their use
 cases without requiring host permissions, and without needing to read the actual requests.
 
+### Differences between [webRequest](/docs/extensions/reference/webRequest/) and [declarativeNetRequest](/docs/extensions/reference/declarativeNetRequest/) API
+
+- The declarativeNetRequest API allows for evaluating network requests in the browser itself. This
+  makes it more performant than the webRequest API, where each network request is evaluated in
+  JavaScript in the extension process.
+- Because the requests are not intercepted by the extension process, declarativeNetRequest removes
+  the need for extensions to have a service worker; resulting in less memory consumption.
+- Unlike the webRequest API, blocking or upgrading requests using the declarativeNetRequest API
+  requires no host permissions when used with the `declarativeNetRequest` permission.
+- The declarativeNetRequest API provides better privacy to users because extensions can't actually
+  read the network requests made on the user's behalf.
+- Unlike the webRequest API, any images or iframes blocked using the declarativeNetRequest API are
+  automatically collapsed in the DOM.
+- While deciding whether a request is to be blocked or redirected, the declarativeNetRequest API is
+  given priority over the webRequest API because it allows for synchronous interception. Similarly,
+  any headers removed through declarativeNetRequest API are not made visible to web request
+  extensions.
```

Comment:
This makes it sound like dNR can be used with MV2. That's because If I'm doing MV3, I can't use webRequest. What am I missing?